pandas_transform/notebooks/dataframe_example.ipynb

```diff
@@ -0,0 +1,2363 @@
+       "...                  ...       ...          ...   ...             ...   \n",
+       "15519       clinical lab  negative      patient  17.0               1   \n",
+       "15520       clinical lab  negative      patient  18.0               1   \n",
+       "15521       clinical lab  negative  other adult  69.0               1   \n",
+       "15522  intl patient svcs  negative  other adult  69.0               0   \n",
+       "15523             ob gyn  negative  other adult  72.0               0   \n",
+       "\n",
+       "       ct_value  orderset payor_group   patient_class  col_rec_tat  \\\n",
+       "0          45.0         0  government       inpatient          1.4   \n",
+       "1          45.0         0  commercial  not applicable          2.3   \n",
+       "2          45.0         1         NaN             NaN          7.3   \n",
+       "3          45.0         1         NaN             NaN          5.8   \n",
+       "4          45.0         1  government       emergency          1.2   \n",
+       "...         ...       ...         ...             ...          ...   \n",
+       "15519      45.0         1         NaN             NaN          3.9   \n",
+       "15520      45.0         1         NaN             NaN          8.0   \n",
+       "15521      45.0         1         NaN             NaN          3.8   \n",
+       "15522      45.0         0  commercial  not applicable          0.8   \n",
+       "15523      45.0         0  commercial  not applicable          0.3   \n",
+       "\n",
+       "       rec_ver_tat  \n",
+       "0              5.2  \n",
+       "1              5.8  \n",
+       "2              4.7  \n",
+       "3              5.0  \n",
+       "4              6.4  \n",
+       "...            ...  \n",
+       "15519          6.8  \n",
+       "15520          5.6  \n",
+       "15521          4.0  \n",
+       "15522          5.2  \n",
+       "15523          6.0  \n",
+       "\n",
+       "[15524 rows x 17 columns]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "covid_testing.loc[:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e104f577-b516-44d8-9723-8678fcdc899e",
+   "metadata": {},
+   "source": [
+    "### Selecting rows that meet a condition\n",
+    "\n",
+    "Let's say we only care about the positive Covid tests. We could make a new DataFrame consisting only of those tests that came back positive using a **conditional** arguement. A condition is a statement that evaluates to either `True` or `False`. \n",
+    "\n",
+    "If, for example, we only want to look at instances where the covid test came back positive, we need to use the condition `covid_testing.loc[:,\"result\"] == \"positive\"`. This statement checks every row in the `covid_testing` and will be `True` for a given row if the entry in columm `result` is equal to the string `positive` and `False` otherwise. "
```

Comment:
Is it worth mentioning that the output of this code is a Series rather than a DataFrame? I can see it either way.